src/main/resources/static/bidder-params/rise.json

```diff
@@ -11,6 +11,10 @@
     "publisher_id": {
       "type": "string",
       "description": "Deprecated, use org instead."
+    },
+    "placementId": {
+      "type": "string",
+      "description": "Placement ID."
```

Comment:
to have better coverage I suggest adding the placement id to the `test-auction-rise-request.json` and `test-rise-bid-request.json` and run the `org.prebid.server.it.RiseTest`, this way it will be clear that placement id is passed through and reach the bidder as-is (if I understand the intention correctly)